src/styles/blocks/container.scss

```diff
@@ -0,0 +1,25 @@
+.container {
+  --column-count: 1;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  padding: 50px 40px 72px;
+  column-gap: 48px;
+  row-gap: 46px;
+  justify-content: center;
+
+  @media (max-width: 488px) {
+    --column-count: 1;
+  }
```

Comment:
you set it above, no need to set it again

Reply:
Not fixed 😢